support.ts

```diff
@@ -1,45 +1,49 @@
 // @ts-ignore
 import { sparqlEscapeUri, sparqlEscapeString } from "mu";
-import fetch from "node-fetch";
-import { LDES_ENDPOINT, LDES_FRAGMENTER } from "./config";
+import { LDES_FOLDER, LDES_FRAGMENTER, } from "./config";
 import { Changeset, Quad, Term } from "./types";
+import { addData, getConfigFromEnv } from "@lblod/ldes-producer";
 
-
+const ldesProducerConfig = getConfigFromEnv();
+console.log("ldesProducerConfig", ldesProducerConfig);
 export function toSparqlTerm(thing: Term): string {
 	if (thing.type == "uri") return sparqlEscapeUri(thing.value);
-	else return sparqlEscapeString(thing.value);
+	// FIXME: without this you probably lose every triples of type boolean & number
+	// a better solution would be to use n3 or something similar
+	else if (thing.datatype?.length) return "\"" + (thing.value?.toString() || "") + "\"" + "^^" + "<" + thing.datatype + ">";
```

Comment:
I don't think this is safe enough. May be better to use the functions available in the javascript template? Here is what we do in lmb: https://github.com/lblod/app-lokaal-mandatenbeheer/blob/master/config/ldes-delta-pusher/publisher.ts#L120
It's not ideal and it would be better if the js template exposed something like this that sparqlEscapes a datatype, but it's a little more safe i think?